utilities/backupable/backupable_db.cc

```diff
@@ -2159,84 +2236,177 @@ Status BackupEngineImpl::BackupMeta::LoadFromFile(
     }
   }
 
+  int schema_major_version = 1;  // implicit default
+
   // Failures handled at the end
```

Comment:
I don't understand the feedback. I'm calling the original schema, with no occurrence of "schema_version" version 1.

Reply:
Sorry just found the comment "// Failures handled at the end" amusing since there's `return`s everywhere (including before your change). It doesn't matter.